browser/resources/settings/brave_overrides/about_page.js

```diff
@@ -3,9 +3,21 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this file,
 // you can obtain one at http://mozilla.org/MPL/2.0/.
 
-import {RegisterPolymerTemplateModifications} from 'chrome://brave-resources/polymer_overriding.js'
+import {html,RegisterStyleOverride,RegisterPolymerTemplateModifications} from 'chrome://brave-resources/polymer_overriding.js'
 import {getSectionElement} from './basic_page.js'
 
+RegisterStyleOverride(
+  'settings-about-page',
+  html`
+    <style>
+      #release-notes {
+        display: block;
+        margin-inline-start: auto;
```

Comment:
Please use `unset` instead of `auto`

Reply:
I have changed to unset and pushed the code kindly review @petemill